packages/sampler/stories/default/ViewManager.js

```diff
@@ -120,22 +119,36 @@ export const _ViewManager = () => {
 	return (
 		<ViewManagerLayout
 			arranger={arranger}
-			duration={number('duration', ViewManagerConfig)}
-			enteringDelay={number('enteringDelay', ViewManagerConfig)}
-			noAnimation={boolean('noAnimation', ViewManagerConfig)}
+			duration={args['duration']}
+			enteringDelay={args['enteringDelay']}
+			noAnimation={args['noAnimation']}
 			onAppear={action('onAppear')}
 			onEnter={action('onEnter')}
 			onLeave={action('onLeave')}
 			onStay={action('onStay')}
 			onTransition={action('onTransition')}
 			onWillTransition={action('onWillTransition')}
-			reverseTransition={boolean('reverseTransition', ViewManagerConfig)}
-			rtl={boolean('rtl', ViewManagerConfig)}
+			reverseTransition={args['reverseTransition']}
+			rtl={args['rtl']}
+			selectedEnd={args['end']}
+			selectedStart={args['start']}
 		/>
 	);
 };
 
+select('arranger', _ViewManager, Object.keys(prop.arranger), ViewManagerConfig, 'SlideBottomArranger');
+number('duration', _ViewManager, ViewManagerConfig);
+number('enteringDelay', _ViewManager, ViewManagerConfig);
+boolean('noAnimation', _ViewManager, ViewManagerConfig);
+boolean('reverseTransition', _ViewManager, ViewManagerConfig);
+boolean('rtl', _ViewManager, ViewManagerConfig);
+select('end', _ViewManager, Object.keys(prop.end), ViewManagerConfig, 'index');
+select('start', _ViewManager, Object.keys(prop.start), ViewManagerConfig, 'index');
+select('direction (SlideArranger)', _ViewManager, prop.direction, ViewManagerConfig, prop.direction[0]);
+range('amount (SlideArranger)', _ViewManager, ViewManagerConfig, {min: 0, max: 100}, 100);
```

Comment:
Can we make these controls like before?

Reply:
I made a separate category for `SlideArranger` in the controls.